Failing test case using image_tag and tag slot (#1885)
ViewComponent/view_component: docs/CHANGELOG.md

## main

> * Add the error `RedefinedExistingMethodError` in case of conflict after naming a slot with the name of an existing method.

Reviewer: ```suggestion
* Raise the error `RedefinedExistingMethodError` when declaring a slot which conflicts with an existing method.
```
